### PR TITLE
Add a note about updating plugin documentation

### LINKED ISSUE #182: [FEATURE]Add a note to the documentation to update doc page when creating or renaming a plugin
We are listing all available OpenSearch and Dashboards plugins on the following documentation pages:

https://opensearch.org/docs/latest/opensearch/install/plugins/#available-plugins
and
https://opensearch.org/docs/latest/dashboards/install/plugins/#available-plugins

Could we add a note to one of the Contributing or Conventions file to either update those pages or create an issue in the [doc repo](https://github.com/opensearch-project/documentation-website/) whenever someone creates a new plugin or updates a plugin name?

### PR DESCRIPTION
**Pull request by kolchfa-aws**
Fixes #182 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by saratvemulapalli** [comment]
Thanks @kolchfa-aws for contributing a change. Do you think this content will best fit in our `STANDARDS.md`[1]?

[1] https://github.com/opensearch-project/opensearch-plugins/blob/main/STANDARDS.md#public-documentation

**Comment by kolchfa-aws** [comment]
@saratvemulapalli This document was suggested by @dblock, so I followed his recommendation. It's really up to you guys where you think it fits in best, but for us on the doc team it's important that the page gets updated so the users have the latest list. Thank you!

**Comment by saratvemulapalli** [comment]
> @saratvemulapalli This document was suggested by @dblock, so I followed his recommendation. It's really up to you guys where you think it fits in best, but for us on the doc team it's important that the page gets updated so the users have the latest list. Thank you!

haha ok. We'll let @dblock decide.

File: BUILDING.md [hunk 1]
```diff
@@ -265,3 +266,8 @@ To exclude add the following in `settings.gradle` file ```startParameter.exclude
     [settings.gradle](https://github.com/opensearch-project/job-scheduler/blob/main/settings.gradle)
     [build.sh](https://github.com/opensearch-project/job-scheduler/blob/main/scripts/build.sh)
 
+## Updating documentation
```
Comment (dblock) [inline BUILDING.md:269]: Should be capital Documentation to match the others

File: BUILDING.md [hunk 2]
```diff
@@ -265,3 +266,8 @@ To exclude add the following in `settings.gradle` file ```startParameter.exclude
...
+
+The available [OpenSearch plugins](https://opensearch.org/docs/latest/install-and-configure/install-opensearch/plugins/#available-plugins) and [Dashboards plugins](https://opensearch.org/docs/latest/install-and-configure/install-dashboards/plugins/#available-plugins) are listed on the documentation site.
```
Comment (dblock) [inline BUILDING.md:271]: Dashboards -> OpenSearch Dashboards

documentation site -> documentation website + link the page

**Review by dblock — CHANGES_REQUESTED** [review]
I have some asks ;)

File: BUILDING.md (hunk 2 above)
Comment (kolchfa-aws) [reply BUILDING.md]: Done

**Comment by saratvemulapalli** [comment]
Few links are broken. We've got to update them. 

```
[STANDARDS.md]:
✗ [404] https://github.com/opensearch-project/anomaly-detection/blob/main/.github/workflows/CI.yml | Failed: Network error: Not Found

[TESTING.md]:
✗ [404] https://github.com/opensearch-project/security-dashboards-plugin/blob/opensearch-1.0.0.0-beta1/.github/workflows/unit-test.yml#L67 | Failed: Network error: Not Found
+ echo

[CONVENTIONS.md]:
✗ [404] https://github.com/opensearch-project/dashboards-notebooks/blob/main/dashboards-notebooks/opensearch_dashboards.json | Failed: Network error: Not Found
```

**Comment by dblock** [comment]
Linked fixed in #193

**Review by dblock — APPROVED** [review]

**Review by saratvemulapalli — APPROVED** [review]

**Pull request merged**